### PR TITLE
Issue 21: Use cBioPortal downloadable data rather than API

### LINKED ISSUE #21: See if sources have downloadable data
Use this rather than hitting API endpoints if possible

### PR DESCRIPTION
**Pull request by korikuzma**
closes #21

### CODE REVIEW

**Comment by korikuzma** [comment]
Not focusing on gnomAD because of this comment: https://github.com/cancervariants/evidence-normalization/issues/21#issuecomment-1081962242

File: evidence/dev/etl/cancer_hotspots.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            data_url, src_dir_path, normalized_data_path, ignore_normalized_data
+        )
+
+    def download_data(self) -> None:
+        """Download Cancer Hotspots data."""
+        if not self.data_path.exists():
+            r = requests.get(self.data_url)
+            if r.status_code == 200:
+                with open(self.data_path, "wb") as f:
+                    f.write(r.content)
```
Comment (jsstevenson) [inline evidence/dev/etl/cancer_hotspots.py:46]: log the code if it isn't 200?

File: evidence/dev/etl/cbioportal.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        super().__init__(data_url, src_dir_path, transformed_mutations_data_path,
+                         transformed_case_lists_data_path, ignore_transformed_data)
+        self.src_dir_etl_path = ETL_DATA_DIR_PATH / "cbioportal"
+        self.src_dir_etl_path.mkdir(exist_ok=True, parents=True)
+        self.msk_impact_2017_dir = None
+
+    def download_data(self) -> None:
+        """Download MSK Impact 2017 data."""
+        response = requests.get(self.data_url, stream=True)
+        if response.status_code == 200:
```
Comment (jsstevenson) [inline evidence/dev/etl/cbioportal.py:59]: also log code if fails

**Review by jsstevenson — CHANGES_REQUESTED** [review]
Might be helpful to retain a little bit more information if downloads fail but looks good otherwise

**Review by jsstevenson — APPROVED** [review]

**Pull request merged**